05-how-to-contribute-to-open-source.md

```diff
@@ -25,139 +29,292 @@ OpenSauced is a powerful tool for finding open source projects to contribute to.
 
-```
-git checkout -b your-branch-name
-```
+   You can create a new branch using the following command:
 
-Replace "your-branch-name" with a descriptive name for your branch, such as "fix-bug-123" or "add-new-feature".
+   ```
+   git checkout -b YOUR-BRANCH-NAME
+   ```
 
-4. **Make your changes**: Now that you have a new branch, you can make changes to the codebase. Be sure to follow the project's coding guidelines and conventions, and test your changes to ensure they work as expected.
+   Replace "YOUR-BRANCH-NAME" with a descriptive name for your branch, such as "fix-bug-123" or "add-new-feature".
 
-5. **Commit your changes**: Once you've made your changes, stage and commit them using the following commands:
+4. **Make your changes**: Now that you have a new branch, you can make changes to the codebase. Always follow the project's coding guidelines and conventions, and test your changes to ensure they work as expected.
 
-```
-git add .
-git commit -m "Your commit message"
-```
+5. **Add and commit your changes**: Once you've made your changes, add your changes to the staging area and commit them with these commands:
 
-Replace "Your commit message" with a brief description of the changes you made.
+   ```
+   git add .
+   git commit -m "Your commit message"
+   ```
+
+   Replace `"Your commit message"` with a brief description of your changes.
 
 6. **Push your changes**: Push your changes to your forked repository on GitHub by running the following command:
 
-```
-git push origin your-branch-name
-```
+   ```
+   git push origin YOUR-BRANCH-NAME
+   ```
 
-Replace "your-branch-name" with the name of your branch.
+   Replace "YOUR-BRANCH-NAME" with the name of your branch.
 
-7. **Submit a pull request**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "New pull request" button. In the "base repository" dropdown, select the original repository, and in the "head repository" dropdown, select your forked repository. Choose your branch from the "compare" dropdown, and click "Create pull request". Fill in the required information and submit your pull request.
+7. **[Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "New pull request" button. In the "base repository" dropdown, select the original repository, and in the "head repository" dropdown, select your forked repository. Choose your branch from the "compare" dropdown. Fill in the required information and submit your pull request by clicking the "Create pull request" button.
```

Comment:
It does stated in the GitHub docs in the link, but I'm not sure if we want to mention "base" and " head" repositories, and comparing branches here as in most cases, we don't need to do that.

I think mentioning those here will confusing beginners.

```suggestion
7. **[Create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork)**: Once you've pushed your changes, navigate to the original project's repository on GitHub and click the "Compare & pull request" button. Fill in the required information and submit your pull request by clicking the "Create pull request" button.
```

Reply:
> It does stated in the GitHub docs in the link, but I'm not sure if we want to mention "base" and " head" repositories, and comparing branches here as in most cases, we don't need to do that.
> 
> I think mentioning those here will confusing beginners.

I agree @adiati98. Base and head confused me when I first learned about the contribution workflow process.